frontend/src/views/ChatView.vue

```diff
@@ -1,3 +1,293 @@
 <template>
-  <div>Chat</div>
+  <v-container>
+    <v-row>
+      <v-col cols="12" md="10">
+        <chat-window
+          v-if="selectedChannel"
+          :channel="selectedChannel"
+          :messages="messages"
+          :users="users"
+          :socket="socket"
+          :key="newMessage"
+          @chat-leave-channel="handleLeaveChannelEvent"
+          @chat-message-menu-selection="handleChatMessageMenuSelection"
+        ></chat-window>
+      </v-col>
+      <v-col cols="12" md="2">
+        <channel-list
```

Comment:
When I resize the page with a width less than 960px, the width of the Channels list expands to take up the entire width of the page and the Drawer disappears. Which is a good thing for our responsiveness.
But when the page is between 960px and 1263px wide, we don't have the Drawer and the Channels list doesn't take the full width of the page. This leads to a weird effect.
However, this is mostly cosmetic, so I don't know if it's a real problem.

Reply:
Opened #162